doc/md: mssql guide (#2527)
ariga/atlas: doc/md/guides/mssql.mdx

To inspect our locally-running SQL Server instance, use the `-u` flag and write the output to a file named `schema.sql`:

```shell
atlas schema inspect -u "sqlserver://SA:P@ssw0rd0995@localhost:1433?database=master&mode=database" --format '{{ sql . }}' > schema.sql
```

Open the `schema.sql` file to view the inspected SQL schema that describes our database.

```sql title="schema.sql"
CREATE TABLE [dbo].[users] (
    [ID] bigint NOT NULL,
    [name] varchar(255) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
    CONSTRAINT [PK_users] PRIMARY KEY CLUSTERED ([ID] ASC)
);
```

</TabItem>
</Tabs>

:::info
For in-depth details on the `atlas schema inspect` command, covering aspects like inspecting specific schemas,
handling multiple schemas concurrently, excluding tables, and more, refer to our documentation
[here](/declarative/inspect).
:::

To generate an Entity Relationship Diagram (ERD), or a visual representation of our schema, we can add the `-w` flag
to the inspect command:

```shell
atlas schema inspect -u "sqlserver://SA:P@ssw0rd0995@localhost:1433?database=master&mode=database" -w
```

![mssql-inspect](https://atlasgo.io/uploads/mssql/images/schema-inspect-hcl.png)

> ## Declarative Migrations
> The declarative approach lets users manage schemas by defining the desired state. Atlas then safely alters the database
> to match. Let's see this in action.

Reviewer: ```suggestion
## Declarative Migrations

The declarative approach lets users manage schemas by defining the desired state of the database as code.
Atlas then inspects the target database and calculates an execution plan to reconcile the difference between the desired and actual states.  Let's see this in action.
```

Do the same for C.H

---
id: getting-started-mssql

> title: Manage MSSQL with Atlas

Reviewer: ```suggestion
title: Manage SQL Server with Atlas
```

Reviewer: Same for ClickHouse

slug: /guides/mssql
tags: [mssql]
---

import InstallationInstructions from '../components/_installation_instructions.mdx';
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> [Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/) (MSSQL) is a powerful relational database management

Reviewer: ```suggestion
[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/) is a powerful relational database management
```

```

Open the `schema.sql` file to view the inspected SQL schema that describes our database.

```sql title="schema.sql"
CREATE TABLE [dbo].[users] (
    [ID] bigint NOT NULL,
    [name] varchar(255) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
    CONSTRAINT [PK_users] PRIMARY KEY CLUSTERED ([ID] ASC)
);
```

</TabItem>
</Tabs>

:::info
For in-depth details on the `atlas schema inspect` command, covering aspects like inspecting specific schemas,
handling multiple schemas concurrently, excluding tables, and more, refer to our documentation
[here](/declarative/inspect).
:::

To generate an Entity Relationship Diagram (ERD), or a visual representation of our schema, we can add the `-w` flag
to the inspect command:

```shell
atlas schema inspect -u "sqlserver://SA:P@ssw0rd0995@localhost:1433?database=master&mode=database" -w
```

![mssql-inspect](https://atlasgo.io/uploads/mssql/images/schema-inspect-hcl.png)

## Declarative Migrations

The declarative approach allows users to manage their database schemas by simply defining the desired end schema, and

> Atlas provides a safe way to alter the database to get there. Let's see this in action.

Reviewer: Something broken about this sentence
